opm/parser/eclipse/Units/UnitSystem.cpp

```diff
@@ -57,6 +57,9 @@ namespace {
         1 / ( Metric::ReservoirVolume / Metric::Time ),
         1 / Metric::Transmissibility,
         1 / Metric::Mass,
+        1, /* gas-oil ratio */
+        1, /* oil-gas ratio */
```

Comment:
Not necessarily, from a quick glance I can only find gas-oil ratio, gas-liquid ratio (uses same unit as gas-oil) and liquid-liquid ratio (water cut). I did choose to include it for some completeness, since oil-gas has a dedicated entry in the manual.